guillotina/traversal.py

```diff
@@ -286,6 +286,7 @@ def __init__(self, resource, request, view):
                 view_result = await self.view()
                 await commit(warn=False)
             except (ConflictError, TIDConflictError):
+                await abort()
```

Comment:
Without adding this, each ConflictError logged in sentry is one connection leaked.

Reply:
okay, so this is fallback to make sure it is completely closed.